app/containers/App/actions.js

```diff
@@ -32,9 +32,35 @@ import {
   ENTITIES_READY,
   ADD_ENTITY,
   UPDATE_ENTITY,
+  DELETE_ENTITY,
```

Comment:
new global actions for creating, deleting and updating entities both remote and local